controllers/suspend_controller.js

```diff
@@ -0,0 +1,41 @@
+const js = require("../data/users.json")
+const list = require("../data/OTP-pass.json")
+const suspend = require("../services/suspendService")
+const fs = require("fs");
+
+
+
+function handleSuspend(request,response) {
+    let body = [];
+    request
```

Comment:
your mixing here between Express and HTTP server approach 
please change it to Express only!